### PR TITLE
feat: add ULID example

### LINKED ISSUE #90: Add ULID
@iuioiua  @kt3k 

when we add ULID to the `deno_std`, let's also add an example here

### PR DESCRIPTION
**Pull request by lino-levan**
Closes #90

### CODE REVIEW

**Comment by lino-levan** [comment]
That script has been so useful. Probably the single best test I've ever written.

File: data/ulid.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * They play very nicely with Deno KV.
+ */
+
+// The standard library contains a function for generating ULIDs.
+import { ulid } from "$std/ulid/mod.ts";
+
+// To generate a ULID, simply call the function.
+console.log(ulid());
+console.log(ulid());
+console.log(ulid());
```
Comment (iuioiua) [inline data/ulid.ts:20]: Nit: perhaps only one of these is needed.

**Review by iuioiua — APPROVED** [review]
LGTM

File: data/ulid.ts (hunk 1 above)
Comment (lino-levan) [reply data/ulid.ts:20]: I think it's important to show that they're all different, but I don't feel strongly about this.

**Comment by iuioiua** [comment]
PTAL @kt3k

**Review by kt3k — APPROVED** [review]
LGTM. Nice!

**Pull request merged**